library/spdm_requester_lib/libspdm_req_get_csr.c

```diff
@@ -143,7 +143,7 @@ static libspdm_return_t libspdm_try_get_csr(libspdm_context_t *spdm_context,
             spdm_context, session_id,
             &spdm_response_size,
             (void **)&spdm_response, SPDM_GET_CSR, SPDM_CSR);
-        if (LIBSPDM_STATUS_IS_ERROR(status)) {
+        if (LIBSPDM_STATUS_IS_ERROR(status) && status != LIBSPDM_STATUS_RESET_REQUIRED_PEER) {
```

Comment:
This addition isn't needed. If the Responder returns an error the flow should go to `receive_done`.

Reply:
Ah you are right. I misunderstood something. I'll update this